src/macros.jl

```diff
@@ -408,11 +410,24 @@ function parse_one_operator_constraint(_error::Function, vectorized::Bool, sense
     parse_one_operator_constraint(_error, vectorized, Val(:in), aff, set)
 end
 
-function parse_constraint(_error::Function, sense::Symbol, lhs, rhs)
+function parse_constraint(_error::Function, ::Val{:call}, sense::Union{Val{:(<=)}, Val{:(≤)}, Val{:(==)}, Val{:(>=)}, Val{:(≥)}, Val{:(.<=)}, Val{:(.≤)}, Val{:(.==)}, Val{:(.>=)}, Val{:(.≥)}}, lhs, rhs)
```

Comment:
Why specifying this union here ? If an extension adds a new `parse_one_operator_constraint`, it needs this method to also work with symbols out of this union.

Reply:
Actually, this union is required to ensure that this function is not called for cases it should not (for instance, for `alldifferent` with two arguments).

Reply:
It seems we should merge the three methods if we replace `lhs, rhs` by `args...`.
Then, instead of creating a custom `parse_constraint` method for `Val{:alldifferent}`, you can create a custom `parse_one_operator_constraint`.